kube/charts/charts.go

```diff
@@ -30,20 +29,28 @@ import (
 	"github.com/docker/compose-cli/kube/charts/kubernetes"
 	chart "helm.sh/helm/v3/pkg/chart"
 	util "helm.sh/helm/v3/pkg/chartutil"
-	helmenv "helm.sh/helm/v3/pkg/cli"
 )
 
 //SDK chart SDK
 type SDK struct {
-	h           *helm.Actions
-	environment map[string]string
+	ctx store.KubeContext
```

Comment:
Do we need the context here, in addition to helmAction pointing at the right cluster / namespace ?